tests/acceptance/test_build.py

```diff
@@ -710,23 +714,89 @@ def test_module_install(
         else:
             originally_on = False
 
-        output = subprocess.check_output(
-            ["debugfs", "-R", "ls -p /usr/share/mender/modules/v3", latest_rootfs]
-        ).decode()
-        entries = [
-            elem.split("/")[5] for elem in output.split("\n") if elem.startswith("/")
-        ]
+        def _check_update_modules_filesystem(rootfs_image, expect_present):
```

Comment:
Q: Check for what?

I would suggest maybe using the expect parameter name to give this some meaning.

Reply:
Renaming to `_check_update_modules_present_in_filesystem` and `_check_update_modules_provided_in_artifact`.